recipes/Fisher-Callhome-Spanish/ST/transformer/train.py

```diff
@@ -625,8 +624,8 @@ def transcription_text_pipeline(transcription):
         valid_loader_kwargs=hparams["valid_dataloader_opts"],
     )
 
-    for dataset in ["dev", "dev2", "test"]:
-        st_brain.evaluate(
-            datasets[dataset],
-            test_loader_kwargs=hparams["test_dataloader_opts"],
-        )
+    # for dataset in ["dev", "dev2", "test"]:
+    #     st_brain.evaluate(
+    #         datasets[dataset],
+    #         test_loader_kwargs=hparams["test_dataloader_opts"],
+    #     )
```

Comment:
Why is it commented now?

Reply:
sorry my bad, I am tuning the conformer recipe, and I simply don't want to evaluate the model after training done.